webapp/src/js/services/branding-images.js

```diff
@@ -72,26 +70,19 @@ angular.module('inboxServices').factory('BrandingImages',
       filter: change => change.id === BRANDING_ID
     });
 
-    var init = updateResources();
+    const init = updateResources();
 
     return {
-      getLogo: function(name) {
+      getLogo: name => {
         if (!name) {
           return '';
         }
-        return getHtml(name);
+        const html = getHtml(name);
+        return html;
       },
-      getAppTitle: function() {
-        return new Promise((resolve, reject) => {
-          DB().get(BRANDING_ID).then(doc => {
-            resolve(doc.title);
-          }).catch(err => {
-            reject(err);
-          });
-        });
-      },
-      replacePlaceholders: function($elem) {
-        init.then(function() {
+      getAppTitle: () => DB().get(BRANDING_ID).then(doc => doc.title),
```

Comment:
You don't need to make a `new Promise` if you already have a promise, so this function becomes a lot cleaner and a little faster too.

The rest of the changes in this file are just ES6 style code for readability.

Reply:
Nice!